src/test/java/uk/gov/dhsc/htbhf/page/BasePage.java

```diff
@@ -56,10 +56,18 @@ public void openDirect() {
     }
 
     public void waitForPageToLoad() {
-        waitForPageToLoad(getPageTitle());
+        if (getPath() != null) {
```

Comment:
could we reverse this logic?
```
if (getPath() == null) {
    waitForPageToLoadWithTitle(getPageTitle());
} else {
    waitForPageToLoadWithPath(getPath());
}
```
It's clearer is we avoid a negative.

Reply:
Done.